examples/nidcpower_source_dc_voltage/README.md

```diff
@@ -27,12 +27,20 @@ This is a MeasurementLink example that sources and measures a DC voltage with an
 
 - NI-DCPower
 
+By default, this example uses a physical instrument. To use a simulated instrument, follow the steps below:
+- Make sure to create a `.env` file in the current directory of the measurement file or any of its parent directories.
+- Add the following environment variables to the `.env` file to set up the simulation:
```

Comment:
```suggestion
By default, this example uses a physical instrument. To use a simulated instrument, do the following:
- Create a `.env` file in the measurement service's directory or one of its parent directories (such as the root of your Git repository or `C:\ProgramData\National Instruments\MeasurementLink\Services`).
- Add the following options to the `.env` file to enable simulation via the driver's option string:
```

Reply:
To make this even easier, why don't we just include a '.env' file in the root directory of each example with our preferred defaults and then tell the user, e.g. to just change
  MEASUREMENTLINK_NIDCPOWER_SIMULATE=0 to 1

and optionally change these settings if they want a different board type or model.
  MEASUREMENTLINK_NIDCPOWER_BOARD_TYPE=PXIe
  MEASUREMENTLINK_NIDCPOWER_MODEL=4141

Reply:
`.env` is owned by the user and is in our `.gitignore` file.

I want to encourage users to share a `.env` file between multiple services.

How about this?
- Move `.env.simulation` to the `examples/` directory.
- Add `> **Note** To enable simulation for all of the examples, copy `.env.simulation` to `.env` in the examples directory.`

Reply:
A related question I haven't been able to find an answer for: If you have multiple .env files in the hierarchy, what is the precedent of the variables in them? E.g. if I have a .env in my example directory and another one in the repo root, both with values for MEASUREMENTLINK_NIDCPOWER_SIMULATE, which one takes precedence?

Reply:
BTW, enabling simulation with `.env` at the root of this Git repo makes some of the tests fail, so it may be better to put `.env.sample` in the `examples/` directory too. My only concern is that the `examples/` directory makes it less visible than keeping it in the root of the Git repo.

Reply:
> A related question I haven't been able to find an answer for: If you have multiple .env files in the hierarchy, what is the precedent of the variables in them? E.g. if I have a .env in my example directory and another one in the repo root, both with values for MEASUREMENTLINK_NIDCPOWER_SIMULATE, which one takes precedence?

Only the first `.env` file is used. There is no merging of `.env` files.

Reply:
> BTW, enabling simulation with .env at the root of this Git repo makes some of the tests fail, so it may be better to put .env.sample in the examples/ directory too.

In case of test case failures, I would also suggest moving the `.env.sample` file to the `examples/` directory. Simultaneously, we can include the below note within the README file located in the examples directory to provide clear instructions:

> **Note** To enable simulation for all of the modular instrument and VISA examples, copy `.env.simulation` to `.env` in the examples directory.

This way, users can easily grasp how to enable simulation and consolidate the configuration into a single .env file, as the instructions are localized within the examples folder for clarity.

Reply:
Sounds good. Adding a note to the README resolves my concern about discoverability.

Reply:
If we relocate the `.env.simulation` file to the `examples/` directory, it's better to also move the .env.sample file to the same directory. Keeping these files in different directories may confuse.

Please let me know if you have different thoughts.

Reply:
Moved both `.env.simulation` and `.env.sample` to the `examples/` directory.